README.md

```diff
@@ -349,14 +349,16 @@ export default Ember.Route.extend(InfinityRoute, {
 
 The `infinity-loader` component as some extra options to make working with it easy!
 
-* **destroyOnInfinity**
+* **hideOnInfinity**
 
 ```hbs
-{{infinity-loader infinityModel=model destroyOnInfinity=true}}
+{{infinity-loader infinityModel=model hideOnInfinity=true}}
 ```
 
-Now, when the Infinity Model is fully loaded, the `infinity-loader` will remove itself
-from the page.
+Now, when the Infinity Model is fully loaded, the `infinity-loader` will hide itself.
+
+***Versions less than 1.0.0 called this property destroyOnInfinity***
```

Comment:
A little weary about changing this just b/c when and if users upgrade, they will have to change this.  However since we are pre 1-0 and semantically this makes more since, I think this is ok.

Reply:
I agree that it is a bummer to break a public API, but should be as simple as find-and-replace all instances of destroyOnInfinity with hideOnInfinity.

Reply:
@hhff you good with this PR or would like any changes made?